tasmanian/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+find_package(Tasmanian 7.5 PATHS "${TASMANIAN_DIR}" REQUIRED)
+
+add_executable(tasmanian_magma example_unstructured_grid.cpp)
```

Comment:
It will compile in any case and it will run just fine with only BLAS or CUDA. Magma is the best option, but it is optional. And even if nothing is enabled, a meaningful message will be given.

The purpose of the example is not to have production code, but documentation in the form of code. So this is good, thanks for checking on it.